### ISSUE TITLE
Correct format for default user project

### ISSUE THREAD

**Issue by wil-den** [alstr/todo-to-issue-action#124]
I have followed (or so I believe) the correct format for the adding of an issue to a default user project but I can see from the build output that it keeps failing. I have added a repository secret and reference it in the workflow file.
Is there any chance you could provide a sample format? I have read your docs and as I mentioned believe I have followed them correctly.

Thanks in advance

**Comment by alstr** [comment]
Hi there. So just to double check, have you:

1. Created a Personal Access Token with `repo` and `workflow` permissions?
2. Created an encrypted secret in your repo, with the value set to this secret key?
3. Added this to your worklow file: `PROJECTS_SECRET: ${{ secrets.NAME_OF_YOUR_SECRET_KEY }}`

An example is here:

https://github.com/alstr/todo-to-issue-action/blob/master/.github/workflows/workflow.yml

You may find it useful to review this issue, where someone was having an issue with projects: https://github.com/alstr/todo-to-issue-action/issues/59

Let me know if you still need help and we'll dig deeper!

**Comment by wil-den** [comment]
Hi there,

Thanks for coming back to me.
So i had not created a PAT and have done so now. 
I am getting a build error that looks like something else:
```
Traceback (most recent call last):
  File "/app/main.py", line 650, in <module>
    last_diff = StringIO(client.get_last_diff())
  File "/app/main.py", line 94, in get_last_diff
    raise Exception('Could not retrieve diff. Operation will abort.')
Exception: Could not retrieve diff. Operation will abort.
```

Maybe it is a state issue. I will try and commit another todo.

Question though: Looking at #59 that you reference, the TOKEN field is specified ${{ secrets.GITHUB_TOKEN}}. In my workflow file I am using the name of the PAT i created to replace GITHUB_TOKEN. Is this correct?

**Comment by wil-den** [comment]
So I realise I made a mistake. The repo and project are part of an ORG that I have. So I have changed from USER_PROJECTS to ORG_PROJECTS. However no matter what variation of ORG_NAME/NAME_OF_PROJECT/COLUMN_NAME I can't get the issue added. I have tried using the project url which includes project number and i get invalid syntax. The project is not a classic project. When I use project name i see an error can't find project.
